tests/generators/ssz_generic/ssz_container.py

```diff
@@ -118,3 +118,20 @@ def invalid_cases():
                                   offset_index=offset_index,
                                   change=lambda x: 0
                               ))
+                        if index == 0:
+                            yield f'{name}_{mode.to_name()}_offset_{offset_index}_minus_one', \
+                                invalid_test_case(lambda: mod_offset(
+                                  b=serialize(container_case_fn(rng, mode, typ)),
+                                  offset_index=offset_index,
+                                  change=lambda x: x - 1
+                                ))
+                        if mode == RandomizationMode.mode_max_count:
+                            serialized = serialize(container_case_fn(rng, mode, typ))
+                            serialized = serialized + serialized[:2]
+                            yield f'{name}_{mode.to_name()}_last_offset_{offset_index}_overflow', \
+                                invalid_test_case(lambda: serialized)
+                        if mode == RandomizationMode.mode_one_count:
+                            serialized = serialize(container_case_fn(rng, mode, typ))
+                            serialized = serialized + serialized[:1]
+                            yield f'{name}_{mode.to_name()}_last_offset_{offset_index}_wrong_byte_length', \
+                                invalid_test_case(lambda: serialized)
```

Comment:
so any particular reason to only add 3 more test types? and not the full combination of `(mode, offset_index)`?

as written, we can probably just pull these three cases out of this loop and specify by hand

if we keep them in the loop then why not just let the loop run and do its thing?

Reply:
>so any particular reason to only add 3 more test types? and not the full combination of

* test 1. support all the test types.
* test 2. really targets  specifically the `mode_max_count` because the overflow
* test 3. targets  specifically the `mode_one_count` because the byte disparity